pkg/osquery/enrollment_details.go

```diff
@@ -31,22 +37,23 @@ func getEnrollDetails(ctx context.Context, osquerydPath string) (service.Enrollm
 		return details, errors.New("Skipping enrollment details")
 	}
 
+	details.OSPlatform = runtime.GOOS
+	details.OSPlatformLike = runtime.GOOS
```

Comment:
Are these the only details we need to be able to avoid the "Unknown Linux Device" issue?

Reply:
I believe so, they are the only details we can "easily" retrieve. I think all the other details require execs or reading various files, basically doing what ever osq does to get the info ... maybe it's not a bad idea to move off of osq completely for enrollment. Definitely expands scope of this task though.

Reply:
I think it might be worth grabbing (via execs) whatever the bare minimum is that we need to avoid "Unknown Linux Device" -- I don't know which details those are, though. Maybe this is already sufficient 🤞

Reply:
This should be it